pyproject.toml

```diff
@@ -38,32 +39,31 @@ hathor-cli = 'hathor.cli.main:main'
 [tool.poetry.dev-dependencies]
 flake8 = "~4.0.1"
 isort = {version = "~5.10.1", extras = ["colors"]}
-yapf = "^0.32"
```

Comment:
We still have yapf's settings in `setup.cfg`.

Should we remove them?

Reply:
Also, would it make sense to start using Black instead? Maybe we could create an issue for this.

Reply:
I forgot about `setup.cfg`, I'll remove yapf's options from there. I think it totally makes sense to start using Black. But I think that CI checks are a must. At some point we stopped using `yapf` and because there was no CI checks we did not realize how much we had drifted. It would also be great to facilitate adding pre-commit hooks. I think we can add that to the issue.

Reply:
Done.